from_cpython/setup.py

```diff
@@ -743,6 +743,10 @@ def detect_modules(self):
             missing.extend(['imageop'])
 
         # readline
+        # Pyston change: in some container system, the dependencies are provided by
```

Comment:
When builds CPython in this kind of environment(a.k.a SlapOS). The build script only needs to provide `LDFALGS` and `CPPFLAGS`. Something like this:

```
CPPFLAGS=-I${zlib:location}/include -I${readline:location}/include ...
LDFLAGS=-L${zlib:location}/lib -L${readline:location}/lib -L${libexpat:location}/lib ...
```